src/content/core/namespaces.mdx

```diff
@@ -59,35 +65,45 @@ To share a namespace go to the Okteto dashboard, select the namespace you want t
+Okteto administrators have the ability to transfer non-personal Namespaces between users. This process can be achieved through the Okteto UI and can be helpful for when you need to manually remove resources from an inactive user. Personal namespaces that are tied to a user are unable to be transferred.
+
+To transfer a Namespace to a new owner, go to the Okteto UI -> Admin -> Namespaces then locate the Namespace that you would like to transfer. Click on the three dots `...` to display the Namespace menu. From the Namespace menu select `Transfer Ownership`.
 
 <p align="center">
   <Image
     src={require("@site/static/img/namespace-transfer-owner.png").default}
-    alt="transfer a namespace"
+    alt="transfer a Namespace"
     width="650"
   />
 </p>
 
-When the Transfer Ownership modal appears, type the email address of the team member who you'd like the transfer the namespace to. Once you press confirm, ownership of that namespace will be immediately transferred.
-If you wish, you can keep the current owner as a member of the namespace by selecting the option in the Transfer Ownership modal. If this is not selected the current owner will immediately lose access.
+When the Transfer Ownership modal appears, type the email address of the team member who you'd like the transfer the Namespace to. Once you press confirm, ownership of that Namespace will be immediately transferred.
+If you wish, you can keep the current owner as a member of the Namespace by selecting the option in the Transfer Ownership modal. If this is not selected the current owner will immediately lose access.
 
 <p align="center">
   <Image
     src={require("@site/static/img/namespace-transfer-modal.png").default}
-    alt="select a new namespace owner"
+    alt="Select a new Namespace owner"
     width="700"
   />
 </p>
 
 :::note
+User specific variables will not transfer along with a Namespace. If the Namespace being transferred has user specific variables, please note that future build or destroy actions may fail.
+
+New user specific variables can be added to a transferred Namespace if desired to avoid future failures. Details on [adding variables can be found here](core/okteto-variables.mdx).
+:::
 
-User specific variables will not transfer along with a namespace. If the namespace being transferred has user specific variables, please note that future build or destroy actions may fail.
+### Garbage collection settings
 
-New user specific variables can be added to a transferred namespace if desired to avoid future failures. Details on [adding variables can be found here](core/okteto-variables.mdx).
+The Okteto Garbage Collector service can automatically scale down idle applications and delete unused namespaces for optimal cost savings and resource management.
```

Comment:
could we crosslink to the definition of the garbage collector and/or idle applications?